test_regress/t/t_vlcov_nfound_bad.out

```diff
@@ -1 +1,4 @@
-%Error: Can't read t/t_NOT_FOUND
+terminate called after throwing an instance of 'std::logic_error'
```

Comment:
These slipped in to this commit, but any coverage test has been failing like this for me. Any idea what's going on here? @wsnyder

Reply:
I don't see that.  Maybe it's memory corruption?  Perhaps --sanitize and/or valgrind will point better at the problem?

Reply:
Thanks I'll look into that. It's just my machine though, it looks like the CI/CD is passing